[ATSPI] Expose `AXAutocompleteValue` accessiblity string attribute to WTR JS

obyknovenius opened the pull request:
#### bab60dac7c2f2c04cf5f995346cb6379c3a187d1
<pre>
[ATSPI] Expose `AXAutocompleteValue` accessiblity string attribute to WTR JS
<a href="https://bugs.webkit.org/show_bug.cgi?id=250374">https://bugs.webkit.org/show_bug.cgi?id=250374</a>

Reviewed by Carlos Garcia Campos.

This attribute is required by `accessibility/custom-elements/autocomplete.html`.

* LayoutTests/platform/gtk/accessibility/custom-elements/autocomplete-expected.txt: Removed.
* Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
(WTR::AccessibilityUIElement::stringAttributeValue):

Canonical link: <a href="https://commits.webkit.org/258780@main">https://commits.webkit.org/258780@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7e5ad9df3a470214b4f97b9fa5ad1b2387356e73

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/102788 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/11908 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/35815 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/112046 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/172269 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/106754 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/12923 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/2817 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/95040 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/109723 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/108563 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/9924 "Build was cancelled. Recent messages:Cleaned up git repository; Skipping applying patch since patch_id isn't provided; Checked out pull request; Pull request contains relevant changes; Skipped layout-tests; 24 flakes 8 failures; 27 flakes 6 failures; Reverted pull request changes; Compiled WebKit (cancelled)") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/93127 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/37570 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/91774 "Build was cancelled. Recent messages:Cleaned up git repository; Skipping applying patch since patch_id isn't provided; Checked out pull request; run-api-tests (cancelled)") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/24648 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/79309 "Build was cancelled. Recent messages:Cleaned up git repository; Skipping applying patch since patch_id isn't provided; Checked out pull request; Updated gtk dependencies; run-api-tests (cancelled)") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/5363 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/26064 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/5512 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/2515 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/11531 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/45557 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/6037 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/7255 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

carlosgcampos reviewed (approved):
LGTM, thanks! but please update the pr and commit message title to match the bug one before merging.

obyknovenius committed:
[ATSPI] Expose `AXAutocompleteValue` accessiblity string attribute to WTR JS
https://bugs.webkit.org/show_bug.cgi?id=250374

Reviewed by Carlos Garcia Campos.

This attribute is required by `accessibility/custom-elements/autocomplete.html`.

* LayoutTests/platform/gtk/accessibility/custom-elements/autocomplete-expected.txt: Removed.
* Tools/WebKitTestRunner/InjectedBundle/atspi/AccessibilityUIElementAtspi.cpp:
(WTR::AccessibilityUIElement::stringAttributeValue):

Canonical link: https://commits.webkit.org/258780@main
